imagestreams/wildfly-centos7.json

```diff
@@ -167,6 +167,26 @@
           "type": "Local"
         },
         "name": "13.0"
+      },
+      {
+        "annotations": {
+          "description": "Build and run WildFly 14 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/openshift-s2i/s2i-wildfly/blob/master/README.md.",
+          "iconClass": "icon-wildfly",
+          "openshift.io/display-name": "WildFly 14",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "sampleRepo": "https://github.com/openshift/openshift-jee-sample.git",
+          "supports": "wildfly:14,jee,java",
+          "tags": "builder,wildfly,java",
+          "version": "14.0"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "docker.io/openshift/wildfly-140-centos7:latest"
+        },
+        "referencePolicy": {
+          "type": "Local"
+        },
+        "name": "14.0"
```

Comment:
please update the "latest" imagestream tag to point to this new tag.  (line 24)

Reply:
Whoops! I forgot about that.

Updated in 745debd11cb233cc0ed73ac0bc21409a18953858.